modules/Microsoft.Sql/servers/databases/deploy.bicep

```diff
@@ -158,26 +158,11 @@ param isLedgerOn bool = false
 @description('Optional. Maintenance configuration ID assigned to the database. This configuration defines the period when the maintenance updates will occur.')
 param maintenanceConfigurationId string = ''
 
-@description('Optional. Differential backup interval in hours.')
-param backupDifferentialInterval int = 24
+@description('Optional. The short term backup retention policies to create for the database.')
+param backupShortTermRetentionPolicies object = {}
 
-@description('Optional. Poin-in-time backup retention in days.')
-@maxValue(35)
-param backupLogRetentionDays int = 7
-
-@description('Optional. Monthly retention in ISO 8601 duration format.')
-param backupWeeklyRetention string = ''
-
-@description('Optional. Weekly retention in ISO 8601 duration format.')
-param backupMonthlyRetention string = ''
-
-@description('Optional. Backup week of year to keep for yearly retention.')
-@minValue(1)
-@maxValue(53)
-param backupYearlyRetentionWeek int = 1
-
-@description('Optional. Yearly retention in ISO 8601 duration format.')
-param backupYearlyRetention string = ''
+@description('Optional. The long term backup retention policies to create for the database.')
+param backupLongTermRetentionPolicies object = {}
```

Comment:
Shouldn't this be singular?

Reply:
Makes sense, I just followed the API naming but we'd never do more that a single deployment per database so singular would fit better.

Reply:
Updated the module now.

Reply:
Thanks, will check in a sec and then we should be good to go.

Regarding the provider you're absoltely right. But I believe it's a convention that its always plural :)